src/models/node.ts

```diff
@@ -57,7 +52,7 @@ const nodeSummary: NodeSummaryModel = {
   blocks: 0,
   network: { height: 0, outbound: 0, inbound: 0 },
   userAgent: "",
-  updateInterval: 1000,
```

Comment:
I don't think this is necessary. The old version used 1000 with no issues, and I like the responsiveness of it. 5 seconds seems slow, and with the increased number of threads on the backend, we should be good.

Reply:
True! I was testing a lot. This can be changed.